docs/docs/userguide/configuration/CONFIGURATION.md

```diff
@@ -130,23 +130,33 @@ instance_types = ["m5.2xlarge"]
 
 The instance type must be a valid [AWS instance type](https://aws.amazon.com/ec2/instance-types/){.external}.
 
-!!! warning "Instance type selection"
+`instance_disk_size` defines the size of default storage attached to an instance.
 
-    The instance type cannot be changed once the infrastructure has been provisioned.
+```terraform
+instance_disk_size = 50
+```
+
+!!! warning "Instance type and disk size selection"
+
+    Both properties cannot be changed once the infrastructure has been provisioned.
 
-### EKS node count
+### Cluster size
 
-`desired_capacity` provides the desired number of nodes that the EKS node group should launch with initially.
+EKS cluster creates an Autoscaling Group (ASG) that has defined minimum and maximum capacity. You are able to set these
```

Comment:
Can turn ASG into a hyperlink to the actual URL where its details are documented i.e. https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler

Reply:
I am not sure, I believe this is talking more about https://docs.aws.amazon.com/eks/latest/userguide/autoscaling.html